fedimint-core/src/db/mod.rs

```diff
@@ -2306,18 +2368,20 @@ pub async fn apply_migrations_dbtx(
         }
 
         while current_db_version < target_db_version {
-            if let Some(migration) = migrations.get(&current_db_version) {
+            if let Some(migration) = migrations.remove(&current_db_version) {
```

Comment:
Does `remove` vs `get` serve any functional purpose? As far as I can tell it does the same thing

Reply:
`get` doesn't require the map to be `mut`, so if possible I think that's a tad better